packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -211,20 +211,29 @@ const ApplicationNavigation = ({
         }}
         className={cx('drawer-menu-focus-trap-container')}
       >
-        <DrawerMenu
-          titleConfig={titleConfig}
-          userConfig={userConfig}
-          hero={hero}
-          navigationItems={navigationItems}
-          activeNavigationItemKey={activeNavigationItemKey}
-          onSelectNavigationItem={onSelectNavigationItem ? generateMenuClosingCallback(onSelectNavigationItem) : undefined}
-          onSelectSettings={onSelectSettings ? generateMenuClosingCallback(onSelectSettings) : undefined}
-          onSelectHelp={onSelectHelp ? generateMenuClosingCallback(onSelectHelp) : undefined}
-          onSelectLogout={onSelectLogout ? generateMenuClosingCallback(onSelectLogout) : undefined}
-          utilityItems={utilityItems}
-          onSelectUtilityItem={onSelectUtilityItem ? generateMenuClosingCallback(onSelectUtilityItem) : undefined}
-          notifications={notifications}
-        />
+        {/*
+          The following div has been added to prevent a "Cannot read property 'ownerDocument' of null" error when using react-intl v5.
+          See link for more info: https://github.com/focus-trap/focus-trap-react/issues/44#issuecomment-543770159
+          This div could be removed with a newer version of FocusTrap
```

Comment:
want to log an issue to remove the div, once we upgrade focus trap?

Reply:
I will log issues to update react-focus trap

Reply:
https://github.com/cerner/terra-application/issues/145
https://github.com/cerner/terra-core/issues/3388
https://github.com/cerner/terra-application/issues/145